bokehjs/src/less/dialogs.less

```diff
@@ -1,11 +1,15 @@
 @import "./_mixins.less";
 
 :host {
-  --bokeh-bg-color: white;
-  --bokeh-border-color: #e5e5e5;
-  --bokeh-shadow-color: #e5e5e5;
+  --bokeh-bg-color: var(--background-color);
+  --bokeh-border-color: var(--divider-color);
+  --bokeh-shadow-color: var(--divider-color);
   --bokeh-ctrl-size: 16px;
-  --bokeh-ctrl-color: gray;
+  // @mattpap - we will definitely need to change this color
+  // to something with better contrast against the lightgray
+  // background color of the dialog header bar. Should we just
+  // change it now?
+  --bokeh-ctrl-color: var(--color);
```

Comment:
I would wait with this this for https://github.com/bokeh/bokeh/discussions/14415.